spec/services/hyrax/lease_manager_spec.rb

```diff
@@ -90,7 +90,7 @@
         expect(manager.lease)
           .to have_attributes visibility_after_lease: 'authenticated',
                               visibility_during_lease: 'open',
-                              lease_expiration_date: an_instance_of(Date),
+                              lease_expiration_date: an_instance_of(DateTime),
```

Comment:
I changed it because on `main`, when you add a lease to a work, it's saved as a DateTime. so I made koppie do the same. idk how the tests weren't failing before. 🤷🏾‍♀️ 

``` bash
[1] pry(main)> ActiveFedora::Base.find('dr26xx36k').lease
=> #<Hydra::AccessControls::Lease id: "0db70bc6-1bd0-4b95-a991-9b4e275b6f33", visibility_during_lease: "open", visibility_after_lease: "restricted", lease_expiration_date: "2023-08-31 00:00:00", lease_history: []>

[2] pry(main)> ActiveFedora::Base.find('dr26xx36k').lease.lease_expiration_date.class
=> DateTime
```